### PR TITLE
Added topic pages for security, LXD, server, desktop and kernel teams

### LINKED ISSUE #39: Create Team landing pages - design
Design the visual elements to denote "team" pages. Marketing have requested that we create landing pages for more specific categories  (aka team pages). For example the 'Maas' landing page would show only Maas-tagged content.
Marketing would also like the visitor to be able to filter results by month and year.

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Added topic pages for security, LXD, server, desktop and kernel teams
- Updated the [copy doc](https://docs.google.com/document/d/1dCX2ll1iAUPFPBNg66Q5wgDYaH2l0-5rs6LK6U5VNWE/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at:
    - [kernel](http://0.0.0.0:8023/topics/kernel/)
    - [security](http://0.0.0.0:8023/topics/security/)
    - [LXD](http://0.0.0.0:8023/topics/lxd/)
    - [server](http://0.0.0.0:8023/topics/server/)
    - [desktop](http://0.0.0.0:8023/topics/desktop/)

## Issue / Card

Fixes #39

### CODE REVIEW

**Comment by anthonydillon** [comment]
Could you remove the card-image class and optionally set the image position. These new hero images would look nicer just vertically centered.

screenshot-2018-1-8 ubuntu insights

File: data/topics.json [hunk 1]
```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "name": "Ubuntu desktop",
```
Comment (anthonydillon) [inline data/topics.json]: Its confusing having desktop as a topic and a category?

File: data/topics.json [hunk 2]
```diff
@@ -34,5 +70,14 @@
         "url": "https://snapcraft.io",
         "logo_url": "https://assets.ubuntu.com/v1/3075aa19-snappy-icon.svg",
         "hero_url": "https://assets.ubuntu.com/v1/a91f36a8-Snapcraft-Visual-Screen.png?w=576"
+    },
+    {
+        "name": "Ubuntu Server",
```
Comment (anthonydillon) [inline data/topics.json]: What is the difference as a user from Cloud & server and the topic server?

File: templates/main-nav.html [hunk 3]
```diff
@@ -40,8 +40,13 @@
       <ul class="hover-menu">
         <li><a href="/topics/design/">Design</a></li>
         <li><a href="/topics/juju/">Juju</a></li>
+        <li><a href="/topics/kernel/">Kernel</a></li>
+        <li><a href="/topics/lxd/">LXD</a></li>
         <li><a href="/topics/maas/">MAAS</a></li>
+        <li><a href="/topics/security/">Security</a></li>
         <li><a href="/topics/snappy/">Snappy</a></li>
+        <li><a href="/topics/desktop/">Ubuntu desktop</a></li>
+        <li><a href="/topics/server/">Ubuntu Server</a></li>
```
Comment (anthonydillon) [inline templates/main-nav.html]: Why do these need to be prefixed with `Ubuntu`? I think its clear they are Ubuntu.

**Comment by spencerbygraves** [comment]
@pmahnke can we swap the security asset please for one without the chain. I've uploaded it to the asset server - https://assets.ubuntu.com/v1/2217d1c8-Security.svg

Thanks.

**Pull request closed without merging**